### PR TITLE
易用性提升已修改的58个API核对 - part2

### PR DESCRIPTION
**Pull request by NKNaN**

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ PyTorch 相比 Paddle 支持更多其他参数，返回参数 Tensor 数量不
 | input   | input        | 输入 Tensor。                                                                                      |
 | bins    | bins         | 直方图 bins(直条)的个数。                                                                          |
 | range   | min, max     | PyTorch 为 bins 的范围，类型为 float，Paddle 为 range 的下边界，上边界，类型为 int，需要转写。 |
-| weight  | -            | 权重，Paddle 无此参数，暂无转写方式。                                                              |
-| density | -            | 结果中每个 bin 是否包含权重数，Paddle 无此参数，暂无转写方式。                                     |
+| weight  | weight       | 权重 Tensor，维度和 input 相同。                                                              |
+| density | density      | 表示直方图返回值是 count 还是归一化的频率，默认值 False 表示返回的是 count。                            |
 | 返回值  | 返回值       | PyTorch 返回 hist 和 bin_edges，Paddle 返回 hist，暂无转写方式。                                   |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 返回值这里，torch返回2个，paddle返回1个，这里新增了一个API：histogram_bin_eges来实现。
所以可以通过`paddle.histogram` 、 `paddle.histogram_bin_eges` 两个API来实现

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ PyTorch 相比 Paddle 支持更多其他参数，返回参数 Tensor 数量不
 ```python
 # PyTorch 写法:
 x = torch.tensor([1., 2, 1])
-y = torch.histogram(x, bins=5, range=(0., 3.))
+y = torch.histogram(x, bins=5, range=(0., 3.))[0]
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 返回值这里可以实现转写

Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 好的

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md (hunk 1 above)
Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 好的，同时新增了 `paddle.histogram_bin_eges` 的中文文档

File: docs/api/paddle/Overview_cn.rst [hunk 3]
```diff
@@ -373,6 +373,7 @@ tensor 线性代数相关
     " :ref:`paddle.cross <cn_api_paddle_cross>` ", "计算 Tensor x 和 y 在 axis 维度上的向量积（叉积）"
     " :ref:`paddle.dist <cn_api_paddle_dist>` ", "计算 (x-y) 的 p 范数（p-norm）"
     " :ref:`paddle.dot <cn_api_paddle_dot>` ", "计算向量的内积"
+    " :ref:`paddle.histogram <_cn_api_paddle_histogram_bin_edges>` ", "返回计算 input 的直方图时所使用的 bins 的边界值 bin_edges"
```
Comment (zhwesky2010) [inline docs/api/paddle/Overview_cn.rst]: API名字写错了

**Comment by NKNaN** [comment]
`paddle.histogram_bin_eges` 的英文文档更新
https://github.com/PaddlePaddle/Paddle/pull/67994

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-## [ 返回参数类型不一致]torch.histogram
+## [ 返回参数类型不一致 ]torch.histogram
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 这个分类是不是应改为 组合实现

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-## [torch 参数更多]torch.Tensor.histogram
+## [ 输入参数用法不一致 ]torch.Tensor.histogram
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md]: 这个分类是不是应改为 组合实现

File: docs/api/paddle/Overview_cn.rst (hunk 3 above)
Comment (NKNaN) [reply docs/api/paddle/Overview_cn.rst]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md (hunk 5 above)
Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md (hunk 4 above)
Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md [hunk 6]
```diff
@@ -9,19 +9,19 @@ torch.Tensor.histogram(bins, *, range=None, weight=None, density=False)
 ### [paddle.Tensor.histogram](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#histogram-bins-100-min-0-max-0)
 
 ```python
-paddle.Tensor.histogram(bins=100, min=0, max=0)
+paddle.Tensor.histogram(bins=100, min=0, max=0, weight=None, density=False)
 ```
 
-PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+其中 PyTorch 的 `range` 与 Paddle 用法不一致，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md]: 这里说明一下需要通过两个API组合实现

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md [hunk 7]
```diff
@@ -9,10 +9,10 @@ torch.histogram(input, bins, *, range=None, weight=None, density=False, out=None
 ### [paddle.histogram](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/histogram_cn.html)
 
 ```python
-paddle.histogram(input, bins=100, min=0, max=0, name=None)
+paddle.histogram(input, bins=100, min=0, max=0, weight=None, density=False, name=None)
 ```
 
-PyTorch 相比 Paddle 支持更多其他参数，返回参数 Tensor 数量不一致，具体如下：
+其中 PyTorch 的 `range` 与 Paddle 用法不一致，且返回参数 Tensor 数量不一致，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 这里说明一下需要通过两个API组合实现

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md [hunk 8]
```diff
@@ -21,20 +21,20 @@ PyTorch 相比 Paddle 支持更多其他参数，返回参数 Tensor 数量不
...
-| density | -            | 结果中每个 bin 是否包含权重数，Paddle 无此参数，暂无转写方式。                                     |
-| 返回值  | 返回值       | PyTorch 返回 hist 和 bin_edges，Paddle 返回 hist，暂无转写方式。                                   |
+| weight  | weight       | 权重 Tensor，维度和 input 相同。                                                              |
+| density | density      | 表示直方图返回值是 count 还是归一化的频率，默认值 False 表示返回的是 count。                            |
+| 返回值  | 返回值       | PyTorch 返回 hist 和 bin_edges，paddle.histogram 返回 hist，paddle.histogram_bin_edges 返回 bin_edges，需要转写。                                 |
 
 ### 转写示例
 
-#### range 参数：bins 的范围
+#### range 参数：bins 的范围；返回值
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 这里分开转写吧，一个转写示例说明一个问题

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md [hunk 9]
```diff
@@ -9,19 +9,19 @@ torch.Tensor.histogram(bins, *, range=None, weight=None, density=False)
...
 ### 参数映射
 
 | PyTorch | PaddlePaddle | 备注                                                                                               |
 | ------- | ------------ | -------------------------------------------------------------------------------------------------- |
 | bins    | bins         | 直方图 bins(直条)的个数。                                                                          |
 | range   | min, max     | PyTorch 为 bins 的范围，类型为 float，Paddle 为 range 的下边界，上边界，类型为 int，需要转写。 |
-| weight  | -            | 权重，Paddle 无此参数，暂无转写方式。   |
-| density | -            | 结果中每个 bin 是否包含权重数，Paddle 无此参数，暂无转写方式。   |
+| weight  | weight       | 权重 Tensor，维度和 input 相同。    |
+| density | density      | 表示直方图返回值是 count 还是归一化的频率，默认值 False 表示返回的是 count。  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md:24]: 返回值写一下 需要转写

转写示例 分开写吧，一个转写示例说明一个问题

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md (hunk 7 above)
Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md (hunk 6 above)
Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md (hunk 9 above)
Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md:24]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md [hunk 10]
```diff
@@ -9,19 +9,19 @@ torch.Tensor.histogram(bins, *, range=None, weight=None, density=False)
 ### [paddle.Tensor.histogram](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#histogram-bins-100-min-0-max-0)
 
 ```python
-paddle.Tensor.histogram(bins=100, min=0, max=0)
+paddle.Tensor.histogram(bins=100, min=0, max=0, weight=None, density=False)
 ```
 
-PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+其中 PyTorch 的 `range` 与 Paddle 用法不一致，需要通过 paddle.Tensor.histogram 和 paddle.Tensor.histogram_bin_edges 组合实现，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md]: 这个写法不太准确，给用户感觉就是 因为range用法不一致，所以需要组合，但其实是因为返回值不同，才需要组合。

直接说明是有两点不同

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md [hunk 11]
```diff
@@ -9,10 +9,10 @@ torch.histogram(input, bins, *, range=None, weight=None, density=False, out=None
 ### [paddle.histogram](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/histogram_cn.html)
 
 ```python
-paddle.histogram(input, bins=100, min=0, max=0, name=None)
+paddle.histogram(input, bins=100, min=0, max=0, weight=None, density=False, name=None)
 ```
 
-PyTorch 相比 Paddle 支持更多其他参数，返回参数 Tensor 数量不一致，具体如下：
+其中 PyTorch 的 `range` 与 Paddle 用法不一致，且返回参数 Tensor 数量不一致，需要通过 paddle.histogram 和 paddle.histogram_bin_edges 组合实现，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md [hunk 12]
```diff
@@ -9,19 +9,19 @@ torch.Tensor.histogram(bins, *, range=None, weight=None, density=False)
...
 ### 参数映射
 
 | PyTorch | PaddlePaddle | 备注                                                                                               |
 | ------- | ------------ | -------------------------------------------------------------------------------------------------- |
 | bins    | bins         | 直方图 bins(直条)的个数。                                                                          |
 | range   | min, max     | PyTorch 为 bins 的范围，类型为 float，Paddle 为 range 的下边界，上边界，类型为 int，需要转写。 |
-| weight  | -            | 权重，Paddle 无此参数，暂无转写方式。   |
-| density | -            | 结果中每个 bin 是否包含权重数，Paddle 无此参数，暂无转写方式。   |
+| weight  | weight       | 权重 Tensor，维度和 input 相同。    |
+| density | density      | 表示直方图返回值是 count 还是归一化的频率，默认值 False 表示返回的是 count。  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md:24]: 下面加一列：返回值

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md (hunk 10 above)
Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md]: 已修改，两点分别说明

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md (hunk 11 above)
Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.histogram.md]: 已修改，两点分别说明

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md (hunk 12 above)
Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histogram.md:24]: 已增加

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Embedding.md [hunk 13]
```diff
@@ -29,8 +31,8 @@ PyTorch 相比 Paddle 支持更多其他参数，具体如下：
 | num_embeddings     | num_embeddings            | 表示嵌入字典的大小。  |
 | embedding_dim     | embedding_dim            | 表示每个嵌入向量的维度。  |
 | padding_idx     | padding_idx            | 在此区间内的参数及对应的梯度将会以 0 进行填充  |
-| max_norm      | -            | 如果给定，Embeddding 向量的范数（范数的计算方式由 norm_type 决定）超过了 max_norm 这个界限，就要再进行归一化，Paddle 无此参数，暂无转写方式。  |
-| norm_type     | -            | 为 maxnorm 选项计算 p-范数的 p。默认值 2，Paddle 无此参数，暂无转写方式。  |
+| max_norm      | max_norm        | 如果给定，Embeddding 向量的范数（范数的计算方式由 norm_type 决定）超过了 max_norm 这个界限，就要再进行归一化。  |
+| norm_type     | norm_type    | 为 maxnorm 选项计算 p-范数的 p。默认值 2.0，Paddle 无此参数，暂无转写方式。  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Embedding.md]: 这个后面还有：`Paddle 无此参数，暂无转写方式。 ` 吗？

Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Embedding.md]: 没有了，已删除

**Review by zhwesky2010 — APPROVED** [review]
LGTM
